### PR TITLE
support-triage: the pattern-1 baseline, a manifest, and a real trace (F-979)

### PR DESCRIPTION
**Pull request by AFFFPupu**
Curriculum **lesson #1**, re-cut off its prompt flaw. This is three of F-979's five build sections; what is still owed is named at the bottom rather than implied.

## 1 · The baseline is now pattern-1

The flaw leaves the system prompt and becomes committed configuration:

```ts
const ISSUE_LOOKUP_TOOLS = [
  "mcp__github__search_issues",
  "mcp__github__list_issues",
  "mcp__github__get_issue",
];
const DENY_ISSUE_LOOKUP = true;   // ← ships as the baseline
```

The prompt it ships with is now the **correct** one — *search the open issues first; only open a new one if nothing already tracks the bug*. The agent follows it. It is refused. It honestly concludes the bug is untracked and files a duplicate.

**That inversion is the whole point.** A stronger model follows the correct instruction *more* reliably and gets refused faster, so the red cannot rot. The old baseline lived in a charter line telling the agent not to search — a pattern-2 flaw whose red is model-dependent and, worse, *indistinguishable from an evaluator that never ran*. That is F-979's own stated rationale, now true rather than planned.

It is also the most common real version of this bug: nobody writes "don't dedup" in a system prompt, and plenty of people ship an agent that cannot see what it needs to.

The managed-agent `v1/v2` YAML pair **stays**, relabelled as what it is — the same story on another runtime through a prompt flaw. Both READMEs say which of the two the curriculum grades.

## 2 · Telemetry

`query` now comes from `@pome-sh/adapter-claude-sdk` instead of the raw SDK. It is a drop-in (the message stream is byte-for-byte what the SDK yields) and it emits gen_ai OTLP spans when a runner injects `POME_OTEL_EXPORTER_OTLP_ENDPOINT` — which both `pome run` and the coach do. The hero example produced the shallowest trace in the catalog; it now carries model, per-turn tokens and latency.

Using the adapter rather than hand-rolling the exporter is deliberate. Per-turn token accounting has two non-obvious traps the adapter already fixed: one API turn arrives as several `assistant` messages that each repeat the same `usage` object (naive counting over-reported a live run's input by 79% — F-994), and the true per-turn `output_tokens` only arrives on a `message_delta` stream event (F-998). An example that re-implemented this would be teaching the bug.

**This moved `@anthropic-ai/claude-agent-sdk` from a pinned `0.2.141` to `^0.3.215`** — the adapter's peer range, and the range `examples/triage-agent` already runs on. Typecheck and tests are clean across the bump; nothing in this examinee's SDK surface changed.

## 3 · Manifest

`pome.json` with `agent.framework` set honestly. support-triage was the only example without one.

## Reviewer notes

* **`test/tool-policy.test.ts` does not assert which way `DENY_ISSUE_LOOKUP` ships**, on purpose — a guard the documented one-line fix turns red is a guard you edit to make green. It earns its place twice: a denial list *reads like ordinary hardening*, so a reader tidying "unused" entries deletes the lesson; and the three names are the twin's **only** read paths to an issue, so a fourth appearing upstream would silently hand the baseline a way around its own defect. There is also a test that the fix never denies the write paths the green side needs — a negative only "the agent did nothing" can satisfy is a trap, not a check.
* **`VERIFICATION.md` now says which baseline each number measured.** The 2026-07 v1/v2 figures are kept **unedited** and dated, with three independent reasons they do not transfer: different flaw, recorded before honest scoring, recorded against pre-2026-08-03 twin images. Deleting the provenance of the "33, not 0" number the whole onboarding narrative rests on would be worse than dating it. The pattern-1 table ships **empty**, with its prediction stated up front so the hosted run can disagree with it.
* **`typecheck:examples` does not cover this package.** It walks the 7 examples with a top-level `package.json`; `support-triage`'s code lives in `local/`. Typechecked manually here — worth a follow-up, not worth widening the script in this PR.

## Still owed — named, not implied

* **The criteria.** They assert the agent *linked the right issue*, not that it *opened no second one* — the negative `docs/curriculum/failure-classes.md` §4.2 asks for, and the one this lesson is named after. It needs `github.no-new-issues` (**F-1198**, #283) to reach the cloud's pin first: **twin-github 0.9.0 merges → publishes → pome-cloud bumps its pin → only then** does the criterion belong in the task file. Added earlier, `criteria-corpus-watch` resolves it against the old pin and reports an unresolved phrase. The README says this out loud rather than leaving a reader to find it.
* **The `verified red` stamp** and the hosted red/green run.
* **The `apps/docs` guide page**, which F-979 says to write *after* the walk so it documents reality.
* **The founder cold walk**, which is the ticket's actual acceptance.

## Verification

```
examples/support-triage/local   typecheck clean · 10 tests pass (2 files)
npm run typecheck:examples      all 7 workspace examples clean
lint:code-health / lint:legacy-markers / lint:copy-markers / lint:cli-pins   pass
```

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: examples/support-triage/pome.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://pome.sh/schemas/v1/pome.json",
+  "agent": {
+    "slug": "support-triage",
+    "name": "Support Triage",
+    "description": "Triages #support bug reports into GitHub issues, and is supposed to notice when one is already tracked",
+    "version": "v1",
+    "framework": "claude-agent-sdk"
+  },
+  "command": "npm --prefix local run start",
```
[the remark this answers is not in the document]
Comment (AFFFPupu) [reply examples/support-triage/pome.json:10]: Confirmed, and it is a CLI bug rather than a manifest bug — filed as **F-1289**.

`cli/src/runner/agentRunner.ts:39-47` spawns with `shell`, `stdio` and `env` and **no `cwd`**, so the command inherits `process.cwd()`. Meanwhile `cli/src/cli/project-config.ts:59-76` finds `pome.json` by walking *up*. The manifest and the command it carries are therefore resolved against two different directories, and `spawningAgentLine` already tells the user the command came "· from pome.json".

The other seven examples dodge this by accident, not by design: their command is `npm start` and their only `package.json` is at the example root, so npm walks up and finds it from any descendant. support-triage is the only one whose runnable package is in a subdirectory.

Not patched here on purpose — the fix belongs in the spawn (cwd := the manifest's directory), which changes behaviour for every example and wants its own test. Measuring it also turned up a second thing worth more than this one: support-triage is **the only example both `examples/` CI gates skip**, for the same root cause (`scripts/typecheck-examples.mjs:29` and `scripts/smoke-examples.mjs:67` key on `examples/<name>/package.json`). That is **F-1290**, and it carries the layout fork — flatten `local/` up, or teach the two scripts a nested package.
